news-bundle/src/Resources/contao/modules/ModuleNews.php

```diff
@@ -369,9 +369,10 @@ protected function generateLink($strLink, $objArticle, $blnAddArchive=false, $bl
 		$strArticleUrl = News::generateNewsUrl($objArticle, $blnAddArchive);
 
 		return sprintf(
-			'<a href="%s" title="%s" itemprop="url">%s%s</a>',
+			'<a href="%s" title="%s"%s itemprop="url">%s%s</a>',
 			$strArticleUrl,
 			StringUtil::specialchars(sprintf($strReadMore, $blnIsInternal ? $objArticle->headline : $strArticleUrl), true),
+			($objArticle->target && !$blnIsInternal ? ' target="_blank" rel="noreferrer noopener"' : ''),
```

Comment:
Shouldn't `rel="noreferrer noopener"` be added to each external link (independent of the target)?

Reply:
That's how it was previously. I assume the thought behind it that there is no real reason to add `noreferrer noopener` for links that point to your own site, as it is not an untrusted link.

Reply:
You could also add an external link and not set it to open in a new window, could you? This is probably what you want for UX anyway.

I know it was like this before but IMHO we should change it. :slightly_smiling_face:

Reply:
@contao/developers WDYT?

Reply:
AFAIK `noopener` only makes sense for `target=_blank` links as links in the same page don’t get an opener reference anyway. I would keep it as it is now.